be/src/util/block_compression.cpp

```diff
@@ -22,6 +22,8 @@
 #include <snappy/snappy-sinksource.h>
 #include <snappy/snappy.h>
 #include <zlib.h>
+#include <zstd.h>
```

Comment:
no need to add some documentation for zstd?

Reply:
It will be better to add some doc for zstd.